src/validations.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-misleading-character-class */
+'use strict'
+// https://v8.dev/features/regexp-v-flag
+// https://mathiasbynens.be/notes/es-unicode-property-escapes
+// https://www.regular-expressions.info/unicode.html
+
+const CHARACTERS = '\\p{L}\\p{M}'
+const INPUT_BASE_REG_EXP = new RegExp(`-0-9.,;:<>?/\n${CHARACTERS}'_ &()/+@%`, 'u')
+const DIGITS = '0-9'
+
+const REGEX_VALIDATION = {
+  NUMBER: new RegExp(`^[${DIGITS}]+$`),
+  EMAIL: /^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/,
+  PATH: /([^/]*)\/*$/,
+  NAME: new RegExp(`^[-${CHARACTERS}' &()/]+$`, 'u'),
+  NAME_WITH_DIGITS: new RegExp(`^[-${CHARACTERS}${DIGITS}' &()/]+$`, 'u'),
+  INPUT: new RegExp(`^[${INPUT_BASE_REG_EXP.source}]+$`, 'u'),
+  ADDRESS: new RegExp(`^[${INPUT_BASE_REG_EXP.source}#]+$`, 'u'),
+  PHONE_CODE: /^(\+)*[0-9]+$/,
```

Comment:
I think + should be mandatory in phone numbers, and we don't know about -, should we include it, or spaces?

Reply:
Code format, for international numbers either starts with `+` sign or `00`: with limit to number of digits. `-` sign is not applicable to phone codes. 
Updated the regex: `^(\+?\d{1,3}|\d{1,4})$`

https://regex101.com/r/dR5ngE/1